### PR TITLE
Resolving bug in HA mode, occurs after network failure between the two nodes

### PR DESCRIPTION
**Pull request by ramindu90**

### CODE REVIEW

File: components/event-processor-manager/org.wso2.carbon.event.processor.manager.core/src/main/java/org/wso2/carbon/event/processor/manager/core/internal/CarbonEventManagementService.java [hunk 1]
```diff
@@ -105,6 +103,9 @@ public void init(HazelcastInstance hazelcastInstance) {
             @Override
             public void memberAdded(MembershipEvent membershipEvent) {
                 checkMemberUpdate();
+                if(haManager!=null){
```
Comment (keizer619) [inline components/event-processor-manager/org.wso2.carbon.event.processor.manager.core/src/main/java/org/wso2/carbon/event/processor/manager/core/internal/CarbonEventManagementService.java]: Code formatting issue. Automatic code formatter will fix this.

File: components/event-processor-manager/org.wso2.carbon.event.processor.manager.core/src/main/java/org/wso2/carbon/event/processor/manager/core/internal/HAManager.java [hunk 2]
```diff
@@ -119,6 +118,18 @@ public void tryChangeState() {
         }
     }
 
+    public void verifyState(){
+        if(activeLockAcquired &&
```
Comment (keizer619) [inline components/event-processor-manager/org.wso2.carbon.event.processor.manager.core/src/main/java/org/wso2/carbon/event/processor/manager/core/internal/HAManager.java]: @ramindu90 
Code formatting issue. Automatic code formatter will fix this.

File: components/event-processor-manager/org.wso2.carbon.event.processor.manager.core/src/main/java/org/wso2/carbon/event/processor/manager/core/internal/CarbonEventManagementService.java (hunk 1 above)
Comment (ramindu90) [inline components/event-processor-manager/org.wso2.carbon.event.processor.manager.core/src/main/java/org/wso2/carbon/event/processor/manager/core/internal/CarbonEventManagementService.java]: Thanks @keizer619, Done in pull request - https://github.com/wso2/carbon-analytics-common/pull/109

File: components/event-processor-manager/org.wso2.carbon.event.processor.manager.core/src/main/java/org/wso2/carbon/event/processor/manager/core/internal/HAManager.java (hunk 2 above)
Comment (ramindu90) [inline components/event-processor-manager/org.wso2.carbon.event.processor.manager.core/src/main/java/org/wso2/carbon/event/processor/manager/core/internal/HAManager.java]: Thanks @keizer619 , Done in pull request - https://github.com/wso2/carbon-analytics-common/pull/109

**Pull request merged**